shortfin/python/shortfin_apps/sd/simple_client.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import json
+import requests
+import argparse
+import base64
+import time
+import asyncio
+import aiohttp
+import sys
+import os
+
+from datetime import datetime as dt
+from PIL import Image
+
+sample_request = {
+    "prompt": [
+        " a cat under the snow with blue eyes, covered by snow, cinematic style, medium shot, professional photo, animal",
+    ],
+    "neg_prompt": ["Watermark, blurry, oversaturated, low resolution, pollution"],
+    "height": [1024],
+    "width": [1024],
+    "steps": [20],
+    "guidance_scale": [7.5],
+    "seed": [0],
+    "output_type": ["base64"],
+    "rid": ["string"],
+}
+
+
+def bytes_to_img(bytes, idx=0, width=1024, height=1024, outputdir="./gen_imgs"):
```

Comment:
Since you have this as an arg from the arg parser no need to have a second default.